var/spack/repos/builtin/packages/py-sierrapy/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PySierrapy(PythonPackage):
+    """A Client of HIVdb Sierra GraphQL Webservice"""
+
+    homepage = "https://github.com/hivdb/sierra-client/tree/master/python"
+    pypi     = "sierrapy/sierrapy-0.3.0.tar.gz"
+
+    maintainers = ['dorton21']
+
+    version('0.3.0', sha256='82474b3815d79d16a480b0cc70b9f7075430ff4990f33306c880b240a3141b6e')
+
```

Comment:
Missing build dep on setuptools, missing a few other dependencies (note that these all need to be exact versions, not version ranges):
```python
certifi==2020.4.5.1
chardet==3.0.4
click==7.1.2
gql==0.4.0
graphql-core==2.3.1
idna==2.9
promise==2.3
requests==2.23.0
rx==1.6.1
six==1.14.0
tqdm==4.46.0
urllib3==1.25.9
voluptuous==0.11.7
```